### PR TITLE
Release cassandra v3.11.4-0.1.0 (automated commit).

### PR DESCRIPTION
**Pull request by mesosphere-teamcity**

### CODE REVIEW

**Comment by mpereira** [comment]
Looks good now @kensipe.

**Comment by guenter** [comment]
nice work! Please add a link to the README in the root of the repo

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Collection of KUDO operators.
 
 The official repository is hosted on Google Cloud Storage under the [Kudo Repository](https://kudo-repository.storage.googleapis.com) bucket.
 
+### [Apache Cassandra](https://github.com/kudobuilder/operators/tree/master/repository/cassandra)
```
Comment (guenter) [inline README.md:9]: Please add a one-sentence description like we have for the other operators.

**Review by guenter — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mpereira) [reply README.md:9]: 5549186267f11a59ca4921caefedfe64e5833989

**Review by guenter — APPROVED** [review]

**Review by kensipe — APPROVED** [review]
🚢 !!

**Pull request merged**